parseIgnored.test.js

```diff
@@ -0,0 +1,36 @@
+import { describe, expect, it } from "vitest";
+import { parseIgnored } from "./index";
+
+describe("parseIgnored", () => {
+    it.each(["", null, undefined, "\r\n", "\n", "#", "#file"])("doesn't ignore when no patterns to ignore provided (%s)", (input) => {
+        // when
+        const isIgnored = parseIgnored(input);
+
+        // then
+        expect(isIgnored("file")).toBe(false);
+        // always ignored
+        expect(isIgnored(null)).toBe(true);
```

Comment:
I think `file.previous_filename` may be undefined diffs that don't rename files? Can you add a test for `isIgnored(undefined)` and handle it in `isIgnored()`?

Reply:
I added a test and luckily it works as `path` is compared with `null` using `==`, not `===` in index.js, line 148 👍🏻 😄 
```javascript
if (path == null || path === "/dev/null") {
```

Reply:
Great :)